ui/src/components/ProvisionerDetailsCard/index.jsx

```diff
@@ -30,8 +30,8 @@ import { ACTION_CONTEXT } from '../../utils/constants';
     fontSize: '0.7rem',
   },
   headline: {
-    paddingLeft: theme.spacing.triple,
-    paddingRight: theme.spacing.triple,
+    paddingLeft: theme.spacing.double,
```

Comment:
Please see the `Migrating to MUI 4.0 PR` this terminology has been depreciated and would result in conflicts after the PR is merged

Reply:
where do i find that to read up

Reply:
You can see #1729 for the replacement

Reply:
but that update has not been added to the master branch. it is still on the previous version of the material ui

Reply:
Correct. We can't use material-ui v4 api at the moment. We will need to rely on `theme.spacing.double` for now.